src/screens/block/block.vue

```diff
@@ -1,5 +1,7 @@
 <template>
-  <div  v-if='block' class="block-screen screen">
+  <div  v-if='block' class="block-screen screen" 
+  v-shortkey="{left: ['arrowleft'], right: ['arrowright']}" 
+  @shortkey='takeAction($event)'>
```

Comment:
Wrong indent: two spaces missed here and on the previous line.
Also you can write just `@shortkey='takeAction'`, without `$event`.